### PR TITLE
Fix script link read&write on IE

### LINKED ISSUE #2776: Read&Write is not reading the script link on IE11 on windows (and incorrectly on Chrome)
**Describe the bug**
Read&Write is not reading the script link on IE11 on windows. On Chrome, on hovering over the script link, all of the content of the Brand component is read. A javascript error is then thrown by the browser extension, and the link is not read again on a second hover:

Screenshot 2019-12-04 at 17 47 33

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to Brand component in psammead storybook
2. Use Read&Write software, and set to 'Read the Web' mode (or 'Hover' mode on chrome)
3. Hover over script link

**Expected behaviour**
Read&Write should read the content of the script link.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by ghost**
Resolves #2776

**Overall change:** _Fix script link read&write on IE._

**Code changes:**

- _Wrap `scriptLink` in a div - This fixes Chrome reading all of the content of the `Brand` component.._
- _Refactor to remove the pseudoelement styling: This fixes the Read&Write issue_.
---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-brand/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 5.1.0-alpha.8 | [PR#2833](https://github.com/bbc/psammead/pull/2833) Wrap `scriptLink` in a div to fix Read&Write issue in chrome |
```
Comment (EinsteinNjoroge) [inline packages/components/psammead-brand/CHANGELOG.md]: We have two entries in the changelog for `5.1.0-alpha.8`, I think we should bump this to `5.1.0-alpha.9` to rectify that

File: packages/components/psammead-brand/src/index.jsx [hunk 2]
```diff
@@ -231,7 +231,7 @@ const Brand = props => {
           <StyledBrand {...props} />
         )}
         {skipLink}
-        {scriptLink}
+        {scriptLink && <div>{scriptLink}</div>}
```
Comment (EinsteinNjoroge) [inline packages/components/psammead-brand/src/index.jsx:234]: We should add snapshot tests that cover this

Comment (ghost) [reply packages/components/psammead-brand/src/index.jsx:234]: This is already covered https://github.com/bbc/psammead/blob/4187ccd1d94d675d7425a74d0812aabe231fbe09/packages/components/psammead-brand/src/index.test.jsx#L24-L79

File: packages/components/psammead-brand/CHANGELOG.md (hunk 1 above)
Comment (ghost) [reply packages/components/psammead-brand/CHANGELOG.md]: Done.

File: packages/components/psammead-brand/src/index.jsx (hunk 2 above)
Comment (EinsteinNjoroge) [reply packages/components/psammead-brand/src/index.jsx:234]: Oh, I see, this covers it. https://github.com/bbc/psammead/blob/4187ccd1d94d675d7425a74d0812aabe231fbe09/packages/components/psammead-brand/src/index.test.jsx#L138-L191

**Review by tochwill — APPROVED** [review]

**Review by EinsteinNjoroge — APPROVED** [review]

**Review by Bopchy — APPROVED** [review]

**Pull request merged**